src/special_functions/Erf.cc

```diff
@@ -0,0 +1,58 @@
+//----------------------------------*-C++-*----------------------------------//
+/*!
+ * \file   special_functions/Erf.cc
+ * \author Kendra Long
+ * \date   Wed Jul 31 08:18:59 2019
+ * \brief  Implementation of Erf approximation
+ * \note   Copyright (C) 2019 Triad National Security, LLC.
+ *         All rights reserved. */
+//---------------------------------------------------------------------------//
+
+#include "Erf.hh"
+#include "units/MathConstants.hh"
+#include <cmath>
```

Comment:
I believe `<cmath>` defines 'erf'; can we just use this? Maybe there are performance concerns.